substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/ResourceFileSystemProvider.java

```diff
@@ -0,0 +1,212 @@
+package com.oracle.svm.core.jdk;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.AccessDeniedException;
+import java.nio.file.AccessMode;
+import java.nio.file.CopyOption;
+import java.nio.file.DirectoryStream;
+import java.nio.file.FileStore;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystemAlreadyExistsException;
+import java.nio.file.FileSystemNotFoundException;
+import java.nio.file.LinkOption;
+import java.nio.file.OpenOption;
+import java.nio.file.Path;
+import java.nio.file.ProviderMismatchException;
+import java.nio.file.ReadOnlyFileSystemException;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.nio.file.attribute.FileAttribute;
+import java.nio.file.attribute.FileAttributeView;
+import java.nio.file.spi.FileSystemProvider;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * ResourcesFileSystemProvider is a core class that supports building a custom file system on top of
```

Comment:
Use doc links for ResourcesFileSystemProvider.

Reply:
supports is superfluous, right? It always builds it.

Reply:
Done.